tests/pom.xml

```diff
@@ -51,14 +51,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>io.openshift.booster</groupId>
-      <artifactId>spring-boot-circuit-breaker-name</artifactId>
-      <version>${project.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>io.openshift.booster</groupId>
-      <artifactId>spring-boot-circuit-breaker-greeting</artifactId>
-      <version>${project.version}</version>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jcl-over-slf4j</artifactId>
```

Comment:
I needed to add this because I was getting NoClassDefFound exceptions without it from RestAssured...
If you have any better ideas than adding this dependency, I would love to hear them

Reply:
This is very weird and I don't remember needing this. But if you need to add it, then maybe add it to the dependencyManagement of the parent to not need the version in here.

Reply:
I will see if there is any way of dropping it and if not, will add it to the `dependencyManagement` section

Reply:
There is definitely something weird going on the dependency chain of rest-assured. I'm trying to get to the bottom of it

Reply:
Finally got to the bottom of this. See the changes in my second commit